Swap mentions of mamba with conda

milljm opened the pull request:
Update our instructions to begin using Miniforge over Mambaforge.

Closes #26322

Commit:
Swap mentions of mamba with conda

Update our instructions to begin using Miniforge over Mambaforge.

Closes #26322

cticenhour commented on modules/doc/content/getting_started/installation/install_miniconda.md:
```diff
@@ -1,40 +1,40 @@
+Follow the steps below depending on your platform to install Miniforge. If you run into issues
 during these steps, please visit our [Conda Troubleshooting](help/faq/conda_issues.md optional=True)
-guide. This installation guide relies on the utilization of `mamba`, an optimized package manager
+guide. This installation guide relies on the utilization of `conda`, an optimized package manager
 for Conda.
```
```suggestion
guide.
```
Conda being an optimized package manager for conda no longer makes sense 😄

Commit:
Remove the dumb

Remove the comment that no longer makes sense.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1943227/) on abdedc1 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/26325/site)

This comment will be updated on new commits.

Commit:
Update Conda after installation

I am finding that before the update, Conda is super slow (still using
the old solver?). After I get conda==23.11.0, installing moose packages
is super quick.
